Feature/shopify-app-cards (#2367)
mercadopago/devsite-docs: guides/shopify/how-tos-antifraude-plus.en.md

# How to prevent fraud in card payments

The **Mercado Pago Antifraude Plus** is an add-on for [Mercado Pago Cards](/developers/en/docs/shopify/integration-configuration/checkout-cards) that optimizes security validation and maintains high approval rates, adding more security by identifying the device ID of buyers and preventing fraud and chargebacks.
To install **Mercado Pago Antifraude Plus**, follow the steps below.
1. From the link received from the Mercado Pago commercial sector, access the **Mercado Pago Antifraude Plus** app page in the "Marketplace" and click on **Install**. If you haven't done so already, log in with your Shopify account.
2. Carefully read the information about the requested permissions and click on **Install** again.
3. If you have already installed the [Mercado Pago Cards](/developers/en/docs/shopify/integration-configuration/checkout-cards) app, you do not need to enter your credentials again, but you can change them if necessary.

> NOTE
>
> Important
> 

> > Remember that, when changing the Shopify password, **it is necessary to renew your credentials**. To do so, follow the instructions in the [Best practices for credentials](/developers/en/docs/shopify/best-practices/credentials-best-practices/secure-credentials) documentation. Then, to update them on your Shopify account,  click on **Manage account** and fill in the corresponding fields with your `access_token` and `public_key` **being careful not to swap the fields when copying and pasting the credentials**.

Reviewer: ```suggestion
> Remember that, when changing the Shopify password, **it is necessary to renew your credentials**. To do so, follow the instructions in the [Best practices for credentials](/developers/en/docs/shopify/best-practices/credentials-best-practices/secure-credentials) documentation. Then, to update them on your Shopify account, click on **Manage account** and fill in the corresponding fields with your `access_token` and `public_key` **being careful not to swap the fields when copying and pasting the credentials**.
```

# How to prevent fraud in card payments

The **Mercado Pago Antifraude Plus** is an add-on for [Mercado Pago Cards](/developers/en/docs/shopify/integration-configuration/checkout-cards) that optimizes security validation and maintains high approval rates, adding more security by identifying the device ID of buyers and preventing fraud and chargebacks.

> 

Reviewer 2: ```suggestion
To install **Mercado Pago Antifraude Plus**, follow the steps below.
```

> 1. From the link received from the Mercado Pago commercial sector, access the **Mercado Pago Antifraude Plus** app page in the "Marketplace" and click on **Install**. If you haven't done so already, log in with your Shopify account.

Reviewer 2: Heits, acá no sería conveniente aclarar previamente que sólo lo van a poder instalar habiendo recibido previamente el link? Una nota o Warning.

Author: @pcanosa Pili, nesse caso não vejo taaanta necessidade porque temos duas formas de instalação. Essa primeira nem será mais relevante em breve, mas como o primeiro contato dos sellers com o app será via e-mail, optamos por deixá-la em primeiro e a versão do ADMIN como a segunda opção (sem a necessidade do link).

> 2. arefully read the information about the requested permissions and click on **Install** again.

Reviewer 2: ```suggestion
2. Carefully read the information about the requested permissions and click on **Install** again.
```

3. If you have already installed the [Mercado Pago Cards](/developers/en/docs/shopify/integration-configuration/checkout-cards) app, you do not need to enter your credentials again, but you can change them if necessary.

> NOTE
>
> Important
> 

> > Renew your credentials as needed, referring to the relevant [documentation](/developers/en/docs/shopify/best-practices/credentials-best-practices/secure-credentials)  as a guide. After renewal, it is essential to update them in the Shopify panel. 

Reviewer 2: ```suggestion
> Renew your credentials as needed, referring to the [documentation](/developers/en/docs/shopify/best-practices/credentials-best-practices/secure-credentials)  as a guide. After renewal, it is essential to update them in the Shopify panel. 
```

> <br><br>

> > Remember: when changing the password, **it is necessary to renew your credentials**. To do this, click on **Manage account** to change your production credentials (`access_token` and `public_key`) in the corresponding fields, **being careful not to swap the fields when copying and pasting the credentials**.

Reviewer 2: ```suggestion
> Remember that, when changing the Shopify password, **it is necessary to renew your credentials**.  To do so,  follow the instructions in the [Best practices for credentials documentation](/developers/en/docs/shopify/best-practices/credentials-best-practices/secure-credentials). Then, to update them on your Shopify account,  click on **Manage account** and fill in the corresponding fields with your `access_token` and `public_key` **being careful not to swap the fields when copying and pasting the credentials**.
```

Reviewer 2: Sería un párrafo donde se condensan las informaciones de los otros dos.
